kiali/README.md

```diff
@@ -85,5 +106,11 @@ When you're done, you can remove the example and all its resources from the clus
 1. Remove the stack by calling Helm:
 
     ```bash
-    helm delete -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME}
+    helm delete -n $KYMA_KIALI_NS $HELM_RELEASE_NAME
+    kubectl -n $KYMA_KIALI_NS delete -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/apirule.yaml
     ```
+
+2. (Optional) If you created the `$KYMA_KIALI_NS` Namespace specifically for this tutorial, remove the Namespace:
+  ```bash
+  kubectl delete namespace $KYMA_KIALI_NS
+  ``` 
```

Comment:
``````suggestion
    ```bash
    kubectl delete namespace $KYMA_KIALI_NS
    ``` 
``````

Ajajaj, it's a bit difficult to check when adding a suggestion, but it seems that I didn't make it a big enough indentation for it to actually take effect...

![image](https://user-images.githubusercontent.com/53601578/194080812-62e1aebf-0406-4ebf-b961-d7d74b3fd835.png)   ➡️ ➡️ ➡️ ➡️    ![image](https://user-images.githubusercontent.com/53601578/194080837-a5ef627e-50f2-48b5-a918-209d92bf0123.png)



This suggestion SHOULD fix it, but please check how the document is rendered after applying changes - it's a good practice :)